libs/plugins/stream-client/src/lib/stage-add/activity-list/activity.component.ts

```diff
@@ -20,12 +19,14 @@ export class ActivityComponent implements Highlightable, OnChanges {
   @Input() activity;
   @Output() selected = new EventEmitter();
   @HostBinding('class.is-active') isHighlighted = false;
+  @Input() iconUrl: string;
   disabled = false;
-  iconUrl: string;
 
   ngOnChanges({ activity: activityChange }: SimpleChanges) {
     if (activityChange) {
-      this.iconUrl = this.activity.icon || ICON_ACTIVITY_DEFAULT;
+      if (this.activity.icon) {
```

Comment:
This is going to override `this.iconUrl` which is marked as `@input()`. I think we either allow passing iconUrl as input or we set iconUrl from `this.activity.icon`. It cannot be both.

Also, what happens if the activity changed but `this.activity.icon` is a falsy value? There should be an else to this if.

Reply:
I looked at the use of this component from `activity-list.component.ts` and the iconUrl is never used an input. I would suggest to remove the input decorator from iconUrl and change this to:

```
if (activityChange) {
  this.iconUrl =  this.activity.icon;
}
```

that way iconUrl would be false if activity.icon is not set